### PR TITLE
Above cm

### PR DESCRIPTION
**Pull request by marcdonovan**

### CODE REVIEW

File: demo/src/demo.js [hunk 1]
```diff
@@ -5,25 +5,26 @@ import CoachMark from '../../main';
...
+		gotIt: true,
 		id: '9834893449'
 	}, function (id) {
 		new CoachMark(document.getElementById('cm-left'), {
-			title: 'Coach Mark Right of Feature',
+			title: 'Coach Mark Below Feature',
 			text: 'Some text explaining to the user why you changed their interface',
 			id: '9834893498'
 		}, function (id) {
 			new CoachMark(document.getElementById('cm-right-1'), {
```
Comment (StommePoes) [inline demo/src/demo.js]: would we be passing dev-torturing down to the next generation by making left mean below and right mean above? Can the id's of the coachmarks be changed to match the new titles/positions instead?

File: src/js/CoachMark.js [hunk 2]
```diff
@@ -82,6 +83,7 @@ export default class CoachMark {
 			contentContainer = document.createElement('div'),
 			paragraph = document.createElement('p'),
 			meatball = document.createElement('div'),
+			gotIt = document.createElement('a'),
```
Comment (StommePoes) [inline src/js/CoachMark.js]: smells like maybe a  button. Can it be a button? Then no useless href="nowhere" needed. I dunno if you're using Elements stuff but there's a class in there for making buttons look like links if you want.

File: src/js/CoachMark.js [hunk 3]
```diff
@@ -55,13 +59,10 @@ export default class CoachMark {
...
-				rect = element.getBoundingClientRect(),
+				rect = element.getBoundingClientRect();
 
-			// 50 is close enough. This is very browser-specific
-				touch_bottom = rect.bottom + 50 + window.pageYOffset > height;
-			// this will follow the 50% rule, but for now, just return bottom
-			// if (touch_bottom) return 'top';
-			return 'bottom';
+			// if the centerline of the element is above the centerline of the viewport, placement is bottom, otherwise top
+			return (rect.bottom - rect.top) / 2 + rect.top > window.innerHeight / 2 ? 'top' : 'bottom';
```
Comment (StommePoes) [inline src/js/CoachMark.js]: nice

File: demo/src/demo.js (hunk 1 above)
Comment (marcdonovan) [inline demo/src/demo.js]: this is actually a div on the left side of the screen, so it still makes sense.

Comment (StommePoes) [inline demo/src/demo.js]: oh ok

File: src/js/CoachMark.js (hunk 2 above)
Comment (marcdonovan) [inline src/js/CoachMark.js]: I changed this and the other links to buttons, but it threw off alignment, so I'll have to fiddle that.

**Comment by StommePoes** [comment]
If you feel your button alignment is good, looks ok to me.

**Comment by wyseguyonline** [comment]
Agreed.

**Pull request merged**